sql_database.go

```diff
@@ -65,20 +65,18 @@ func makeRowOutputter(w *SafeCSVWriter, r *sql.Rows) (*rowOutputter, error) {
 
 func (ro *rowOutputter) outputRows(r *sql.Rows) error {
 	if err := r.Scan(ro.pointers...); err != nil {
-		// Since rows are output to a CSV as strings, there is no possible output
-		// string that uniquely represents NULL and not a legitimately possible
-		// string value.
-		//
-		// When the output value is a string, and the row value is NULL/nil,
-		// database/sql.convertAssign will return this error.
-		//
-		if strings.HasSuffix(err.Error(), "unsupported Scan, storing driver.Value type <nil> into type *string") {
-			err = fmt.Errorf("%v, NULL cannot be uniquely encoded as a CSV value, consider using the SQL function COALESCE to convert NULLs to some default value", err)
-		}
 		return err
 	}
 
-	if err := ro.w.Write(ro.values); err != nil {
+        vals := make([]string, len(ro.values))
+        for i,v := range ro.values {
+            if v.Valid {
+                vals[i] = v.String
+            } else {
+                vals[i] = "NULL"
```

Comment:
I prefer "\N" instead of "NULL", since it is the preferred for mysql load data:

> https://dev.mysql.com/doc/refman/5.7/en/load-data.html

In the fullness of time, this should be a database specific log inside database.go, or a parameter/config option.